dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Converters/GridFormingControlDispatchableVirtualOscillatorControl.mo

```diff
@@ -110,17 +110,17 @@ equation
     Line(points = {{-58, -1}, {-50, -1}, {-50, 23}, {-46, 23}}, color = {0, 0, 127}));
   connect(iqConvPu, virtualImpedance.iqConvPu) annotation(
     Line(points = {{-58, -6}, {-49, -6}, {-49, 5}, {-46, 5}}, color = {0, 0, 127}));
-  connect(udFilterPu, voltageLoop.udFilterPu) annotation(
+  connect(udFilterPu, voltageLoopudFilterPu) annotation(
```

Comment:
pb here (I think there are more of these)
You should have replaced " p.u" and " p.u." with " pu" instead (with a space before)